README.md

```diff
@@ -97,25 +106,25 @@ We have worked on the adding following core features and functionality:
 ## Steps to Deploying on Heroku
 
 - ... create a heroku app
+- `heroku stack:set heroku-20` (Ruby 2.7.7 is not supported on latest stack heroku-22. Double check your Ruby version though)
```

Comment:
```suggestion
```

This should no longer be necessary with Ruby 3 as the default. :)

Reply:
But we are using 2.7.7 though?

Reply:
oh, ignore me! I'm getting confused. (only snap con was recently updated...the ruby 3 stuff for teacher tracker isn't done...)